gammapy/estimators/points/core.py

```diff
@@ -367,13 +379,24 @@ def from_table(
                 )
 
         meta = cls._get_meta_gadf(table)
-        return cls.from_maps(
-            maps=maps,
-            reference_model=reference_model,
-            meta=meta,
-            sed_type=sed_type,
-            gti=gti,
-        )
+
+        if format == "lightcurve":
+            return LightCurve.from_maps(
```

Comment:
Is this the responsibility of the parent class?

Reply:
The idea here was not only to give "responsability" to the parent, but to create a Lightcurve "by force" when using FluxPoints with a time axis (e.g. to keep compatibility with old code but still access the new functionalities and checks)